ansible_base/utils/validation.py

```diff
@@ -101,6 +104,23 @@ def validate_cert_with_key(public_cert_string, private_key_string):
     return True
 
 
+def validate_image_data(data: str) -> None:
+    # in case we are passed an empty string, we can skip validation
+    if not data:
+        return None
+
+    CUSTOM_LOGO_RE = re.compile(r'^data:image/(?:png|jpeg|gif);base64,([A-Za-z0-9+/=]+?)$')
+
+    match = CUSTOM_LOGO_RE.match(data)
+    if not match:
+        raise ValidationError("Invalid format for custom logo. Must be a data URL with a base64-encoded GIF, PNG or JPEG image.")
```

Comment:
Wrap this string and the one below in the translation module `_()`. See line 95 as an example.